app/models/mhv_account.rb

```diff
@@ -120,8 +127,12 @@ def create_mhv_account!
 
   def upgrade_mhv_account!
     if may_upgrade?
+      StatsD.increment("#{STATSD_ACCOUNT_UPGRADE_KEY}.total")
+
       client_response = mhv_ac_client.post_upgrade(params_for_upgrade)
       if client_response[:status] == 'success'
+        StatsD.increment("#{STATSD_ACCOUNT_UPGRADE_KEY}.success")
+
```

Comment:
hmmm, so with upgrade things might get a little weird.

We make the call to upgrade when the state of an account is unknown, in other words the user already had an mhv account, but it may or may not be premium (we just don't know).

Do we still want to log this as part of the total? I think maybe we would want to update the total and success upon success. And additionally increment the total when upgrade fails for any reason other than "account already premium"